.gitignore

```diff
@@ -112,3 +112,7 @@ license/
 vonk-trial-license.json
```

Comment:
@liberaliscomputing Can you do me a favor and remove the vonk license and the two settings.env files here?

Reply:
Deleted!